main.py

```diff
@@ -25,26 +27,28 @@
     if query == 'q':
         break
 
-    dbdoc = find_DB.find_match(query)
-    response = find_DB.get_response(dbdoc)
-
-    course_reqd = string_functions.find_match(user.courses, query)
-
-    if response and not course_reqd:
-        print
-        print response
-        print 
-        continue
-
-    print
-
-    keyword = string_functions.get_keyword(query)
-
-    response = handle_query.get_response(keyword, course_reqd, user)
-
-    if response is None:
-        print 'can\'t get you'
-    else:
-        print response
-
-    print
+    # dbdoc = find_DB.find_match(query)
```

Comment:
Remove comments that are redundant. If it is useful in future for debugging we can always lookup previous commits.

Reply:
this looks like an older file. they've been removed in the later versions